app/assets/stylesheets/application.scss

```diff
@@ -12,10 +12,12 @@
  *
  */
 @import "bootstrap-sprockets";
+@import "bootstrap";
```

Comment:
So you're envisioning that it's really just for use for overriding bootstrap variable values?